app/bundles/AssetBundle/Entity/Download.php

```diff
@@ -33,17 +33,17 @@ class Download
     private $ipAddress;
 
     /**
-     * @var \Mautic\LeadBundle\Entity\Lead
+     * @var \Mautic\LeadBundle\Entity\Lead|null
      */
     private $lead;
 
     /**
-     * @var string
+     * @var int
```

Comment:
Should this be nullable as well?

Reply:
I see. We are going to use rector to move the required fields to a constructor so this should not be null. Just saw the comment on the other PR.

Reply:
@diaboloshogunate We will use rector, I tried it before on but it was not brilliant with job. I am thinking of better ways to build annotation so rector can pick it up

Reply:
@diaboloshogunate rector will pick the obvious cases, so we need to make changes to make it rector compatible